test/unit/floats.js

```diff
@@ -0,0 +1,82 @@
+var test = require('tap').test,
+nodeo = require(__dirname + '/../../lib/nodeo.js');
+
+var CF = nodeo.chromosome_float;
+var Classic = nodeo.classic_float;
+
+test('loads', function (t) {
+         t.ok(Classic, 'Loaded OK');
+         t.end();
+     });
+
+var population_size = 16,
+population = new Array([]),
+chromosome_size = 16;
+
+
+test('chromosomes floats', function (t) {
+    var tmpChr=[];
+    for( var i=0; i<chromosome_size; ++i ) tmpChr[i]=Math.random();
+    for ( var i = 0; i < population_size; i ++ )  {
+	var chromosome = new CF.Chromosome( tmpChr );
+	population.push( chromosome );
+	t.equal(chromosome.vector.length, chromosome_size, "Length " + chromosome.vector.toString() + " OK" );
+	/*var is_match = chromosome.string.match(/[01]+/g );
+	t.ok(is_match, "Binary");
+        chromosome.fitness =  nodeo.utils.max_ones( chromosome.string );
+        */
+	t.ok( chromosome.fitness >= 0, "fitness" );
+
+    // Testing mutation
+	var new_chromosome = CF.mutate(chromosome);
+	t.ok( !new_chromosome.vector.reduce(function(prev,e,i){return prev && e==chromosome.vector[i];},true), // Compares vector position by position
+         "mutation " + chromosome.vector.toString() + " - " + new_chromosome.vector.toString());
+	t.equal( new_chromosome.vector.length, chromosome.vector.length, "Length mutation" );
+	if ( nodeo.utils.sum_ones( new_chromosome.vector )<chromosome_size/2 ) {
+        // Testing invert
+	    var to_cross = CF.invert( chromosome );
+	    t.ok( to_cross.vector.reduce(function(prev,e,i){return prev && e==to_cross.vector[i];},true) !== chromosome.vector.reduce(function(prev,e,i){return prev && e==to_cross.vector[i];},true)
+            , "invert " + to_cross.vector.toString() + " - " + chromosome.vector.toString() );
+
+        // Testing crossover
+	    var crossed = CF.crossover( to_cross, chromosome );
+	    t.ok( !crossed[0].vector.reduce(function(prev,e,i){return prev && e==to_cross.vector[i];},true),  // Compares vector position by position
```

Comment:
Line is too long.<br>Identifier 'to_cross' is not in camel case.<br>Don't make functions within a loop.